### PR TITLE
Chimoney Whatsapp Bot using Nodejs

### PR DESCRIPTION
**Pull request by amosayomide05**
This bot allows a whatsapp user to send chimoney to another user inside a whatsapp group.

### CODE REVIEW

**Comment by amosayomide05** [comment]
#299

**Comment by amosayomide05** [comment]
@uchibeke

**Review by uchibeke — CHANGES_REQUESTED** [review]
Could you add a readme and document the steps to start this bot?

Also, please record a screen video of your phone/laptop showing this bot in action. Would make it easier to review and replicate.

Thanks

**Comment by amosayomide05** [comment]
Send the request

**Comment by amosayomide05** [comment]
The bot sends claim link to the receiver

**Comment by amosayomide05** [comment]
To run the bot

1. `npm i`
2. `node index`

**Comment by amosayomide05** [comment]
I have added a readme

File: submissions/whatsapp-bot-node/readme.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+The Environment variables (.env) file contains:
+
+- `PORT`: Port on which the bot's HTTP server will run.
+- `BOT_URL`: Your whatsapp bot's URL.
+- `API_KEY`: Your API key for Chimoney integration.
+- `REDIRECT_URL`: Redirect URL for Chimoney transactions.
+- `CHIMONEY_BASE_URL`: Base URL for Chimoney operations.
+- `CHIMONEY_BASE_HOST`: Chimoney base host URL.
+- `DEFAULT_EMAIL`: Default email to receive transactions updates (since telegram doesn't have emails).
```
Comment (uchibeke) [inline submissions/whatsapp-bot-node/readme.md:12]: What is this used for? You mention Telegram

Comment (uchibeke) [reply submissions/whatsapp-bot-node/readme.md:12]: Is this a Whatsapp or Telegram bot?

Comment (amosayomide05) [reply submissions/whatsapp-bot-node/readme.md]: I made a mistake, it's whatsapp

**Review by uchibeke — APPROVED** [review]

**Comment by amosayomide05** [comment]
@phyleria

**Pull request merged**